index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "react-native-get-location" {
+  export class Location {
+    latitude: number;
+    longitude: number;
+    altitude: number;
+    accuracy: number;
+    speed: any;
+    time: number;
+    bearing?: number;
+    provider?: number;
+    verticalAccuracy?: number;
+    course?: number;
+  }
+  export interface GetCurrentPositionOptions {
+    enableHighAccuracy: boolean;
+    timeout: number;
+  }
+
+  export class LocationError extends Error {
+    name: string;
+    code: string;
+  }
+
+  export function getCurrentPosition(
+    options: GetCurrentPositionOptions
+  ): Promise<Location>;
```

Comment:
Here we need to define the error?

```suggestion
  ): Promise<Location, LocationError>;
```

Reply:
Going to go check, if the error is returned (not thrown) by the `getCurrentPosition` function then yes I should have put it in the Promise. Let me go check.

Reply:
The error is thrown.